.github/workflows/testing.yml

```diff
@@ -0,0 +1,29 @@
+name: Testing
+
+on:
+  push:
+    branches:
+    - master
+  pull_request:
+
+jobs:
+  build:
+    name: Build OpenGFX
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Set up Python 3.8
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.8
+    - name: Install dependencies
+      run: |
+        sudo apt install -y gimp --no-install-recommends
+        python -m pip install --upgrade pip
+        # TODO -- If nml has a new release (higher than 0.4.5), this can just be 'pip install nml'
+        python -m pip install https://github.com/OpenTTD/nml/archive/master.zip
+    - name: Build
+      run: |
+        make Makefile.vcs Makefile.gfx
```

Comment:
Why is this extra rule necessary? By my reading, they should be built by the standard make rules...

Reply:
Yeah ... current Makefile is a bit funky. It first includes these files, and later generates them. This gives both warnings, and wrong variable-values.

So a quick way to solve this is to generate them before running the rest.

Makefile needs some serious attention ;)

Reply:
I thought that including makefiles like that was fine - it should just do a file/rule check same as any other rule. What warnings are you seeing?

Reply:
```
Makefile:158: Makefile.vcs: No such file or directory
Makefile:256: Makefile.gfx: No such file or directory
```
I am sure it can be fixed one way or the other, but I rather first get this to work, and look into these details later. I will add a note how-ever, as that is currently missing.